src/EditorFeatures/Core/Implementation/Suggestions/SuggestedAction.cs

```diff
@@ -86,7 +90,39 @@ protected Task<ImmutableArray<CodeActionOperation>> GetPreviewOperationsAsync(Ca
                 async () => await CodeAction.GetPreviewOperationsAsync(cancellationToken).ConfigureAwait(false), cancellationToken);
         }
 
-        public virtual void Invoke(CancellationToken cancellationToken)
+        public void Invoke(CancellationToken cancellationToken)
```

Comment:
This code still makes me squeamish :frowning:.  @jasonmalinowski - can you take a look and either reassure me or suggest something better to do?